README.md

```diff
@@ -73,6 +73,10 @@ settings do
   # various others...
   provide "solr_writer.commit_on_close", "true"
 
+  # Basic authentication is configurable if needed.
+  provide "solr_writer.basic_auth_user", "foo"
+  provide "solr_writer.basic_auth_password", "bar"
+
```

Comment:
I think maybe the README is long enough, and we can leave this out of the README to be found in the settings doc where you have also added it (thanks!) -- apparently nobody has needed it until now, so I'd consider it a lesser needed/used feature. What do you think?

Reply:
Make sense.  I'll remove it.